src/main/java/edu/wpi/grip/core/operations/composite/ContoursReport.java

```diff
@@ -0,0 +1,25 @@
+package edu.wpi.grip.core.operations.composite;
+
+import static org.bytedeco.javacpp.opencv_core.*;
+
+/**
+ * The output of {@link FindContourOperation}.  This stores a list of contours (which is basically a list of points) in
+ * OpenCV objects, as well as the width and height of the image that the contours are from, to give context to the
+ * points.
+ */
+public final class ContoursReport {
+    protected int rows = -1, cols = -1;
+    protected MatVector contours = new MatVector();
+
```

Comment:
> I think the report classes should be moved to a separate package.

Yeah, I kind of agree we should overall revamp the package structure a little.  `operations` should probably not be in `core`, `Source` should be in the `sources` package, etc...